apps/src/templates/JsDebugger.jsx

```diff
@@ -45,17 +45,17 @@ var DebugButtons = function () {
   return (
     <div id="debug-commands" className="debug-commands">
       <div id="debug-buttons">
-        <span> </span>
+        {" " /* Explicitly insert whitespace so that this behaves like our ejs file*/}
         <button id="pauseButton" className="debugger_button">
           <img src="/blockly/media/1x1.gif" className="pause-btn icon21"/>
           {i18n.pause()}
         </button>
-        <span> </span>
+        {" " /* Explicitly insert whitespace so that this behaves like our ejs file*/}
         <button id="continueButton" className="debugger_button">
```

Comment:
Yeah, in a lot of cases it's probably fine to remove, but it will result in a slightly different render. If we're going to do that, I'd like to do it in a separate pass (i.e. pass 1 is lots of code change, but eyes tests will pass because things render identically, pass 2 is minimal code change with eyes tests failing because things render slightly differently).